doc/man7/EVP_KDF-KB.pod

```diff
@@ -140,7 +140,7 @@ Label "label", and IV "sixteen bytes iv".
  *p = OSSL_PARAM_construct_end();
  if (EVP_KDF_CTX_set_params(kctx, params) <= 0)
      error("EVP_KDF_CTX_set_params");
- else if (EVP_KDF_derive(kctx, out, sizeof(out)) <= 0)
+ else if (EVP_KDF_derive(kctx, out, sizeof(out), NULL) <= 0)
```

Comment:
This should probably remove the set_params line and do this instead..
````
if (EVP_KDF_derive(kctx, out, sizeof(out), params) <= 0)
    error("EVP_KDF_derive");
````

Reply:
Yeah, what @slontis wrote.

Reply:
I understand the intention was to show the two possible API. Either call `EVP_KDF_CTX_set_params` or call `EVP_KDF_derive` with params.
Calling `EVP_KDF_derive` with params is already shown in the sample just above.

Reply:
relates to commit 6980e36a2aab7a916e8bdcdb70ee03ebaa1bc1cf made by @paulidale.
Just want to make sure.

Reply:
I think the best example would be to have the params going to the derive call and to not have a separate set_params call.  This example **will** be copy/pasted into applications, it should be done "properly".  I.e. params in derive rather than separately.

I missed this in my changes, that's all.

Reply:
I would weight in favor on dropping `set_params` altogether then.

Reply:
It's an argument yet to come....

Reply:
I pushed an updated version without `set_params`. I'll also update the  PR I have against the rust binding.

Reply:
Did you remember to push your update? Is still see the `set_params` call?

Reply:
pebkac :) thanks for the reminder.